lws_config.h.in

```diff
@@ -17,6 +17,12 @@
 /* The Libwebsocket version */
 #cmakedefine LWS_LIBRARY_VERSION "${LWS_LIBRARY_VERSION}"
 
+#define LWS_LIBRARY_VERSION_MAJOR ${LWS_LIBRARY_VERSION_MAJOR}
+#define LWS_LIBRARY_VERSION_MINOR ${LWS_LIBRARY_VERSION_MINOR}
+#define LWS_LIBRARY_VERSION_PATCH ${LWS_LIBRARY_VERSION_PATCH}
+/* LWS_LIBRARY_VERSION_NUMBER looks like 1005001 for e.g. version 1.5.1 */
+#define LWS_LIBRARY_VERSION_NUMBER (LWS_LIBRARY_MAJOR*1000000)+(LWS_LIBRARY_VERSION_MINOR*1000)+LWS_LIBRARY_VERSION_PATCH
```

Comment:
So when doing a release, you change the major/minor version (2.1.3 -> 2.2.3), then set the Release flag to `0`, tag the release and so on.

Then in the next commit you set the release flag to `1`. So the version will still be the same (2.2.3) however, when comparing this to the release version, it will show up as GREATER than, because of that last "alpha/beta"-bit :)